### ISSUE TITLE
Bug: Timestamp difference between Time and Time::createFromTimestamp

### ISSUE THREAD

**Issue by JamesShaver** [codeigniter4/CodeIgniter4#5594]
### PHP Version

7.4

### CodeIgniter4 Version

4.1.7

### CodeIgniter4 Installation Method

Composer (using `codeigniter4/appstarter`)

### Which operating systems have you tested for this bug?

Linux

### Which server did you use?

fpm-fcgi

### Database

MySQL 5.7.36

### What happened?

Time and Time::createFromTimestamp produce differing results:

- 2022-01-17 13:57:53
- 2022-01-17 19:57:53

### Steps to Reproduce

```
 namespace App\Controllers; 
 use CodeIgniter\I18n\Time;
 class Timetest extends BaseController
 {
    public function index()
    {
        $now = new Time('now', 'America/Chicago', 'en_US');
        $new_now = Time::createFromTimestamp($now->timestamp, 'America/Chicago', 'en_US');

        print $now;
        echo "<br>\n";
        print $new_now;
    }
} 
```

### Expected Output

I expect the timestamps to match.

### Anything else?

Submitting this issue because of the forum thread [here](https://forum.codeigniter.com/thread-81033.html)

**Comment by paulbalandan** [comment]
@JamesShaver thanks for the report. Can you check if this will be fixed by #5588 ?

**Comment by JamesShaver** [comment]
Thanks @paulbalandan, I did search for the issue but missed this.  We can close my report.
